From d6891bc80e1510faf32f7271546a430377da5a09 Mon Sep 17 00:00:00 2001
From: Ruslan Bilovol <ruslan.bilovol@ti.com>
Date: Wed, 14 Aug 2013 11:35:48 +0300
Subject: [PATCH 17/17] ARM: dts: omap4-var-som: configure connection to PMIC
 on Variscite's SOM

Now that typical OMAP4 to PMIC connection is defined
in a separate .dtsi file, ve can configure properly OMAP4
pins connected to TWL6030 by just including it.

Include that configuration file for Variscite's SOM.

Signed-off-by: Ruslan Bilovol <ruslan.bilovol@ti.com>
Signed-off-by: Benoit Cousson <bcousson@baylibre.com>
---
 arch/arm/boot/dts/omap4-var-som.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/omap4-var-som.dts b/arch/arm/boot/dts/omap4-var-som.dts
index b41269e..6f0b763 100644
--- a/arch/arm/boot/dts/omap4-var-som.dts
+++ b/arch/arm/boot/dts/omap4-var-som.dts
@@ -40,6 +40,7 @@
 };
 
 #include "twl6030.dtsi"
+#include "twl6030_omap4.dtsi"
 
 &i2c2 {
 	clock-frequency = <400000>;
-- 
1.8.4.rc3

